### ISSUE TITLE
QAT conv layer with bias

### ISSUE THREAD

**Issue by dtcrong** [pytorch/pytorch#28832]
I have a resnet-based network. Trying to convert it to INT8 but met accuracy loss using static quant.
Tried QAT but met with assert error "Only support fusing Conv2d that does not have bias"

Look into the model structure and found all the conv layers have bias:
nn.Conv2d(in_planes, out_planes, kernel_size, stride=stride, bias=True)

is it possible to use fake quant to train it?

**Comment by soumith** [comment]
please use https://discuss.pytorch.org for questions. You can tag the questions with `quantization` tag
